### ISSUE TITLE
Gradle build failed with unknown error

### ISSUE THREAD

**Issue by tomgiam** [expo/eas-cli#1210]
### Build/Submit details page URL

https://expo.dev/accounts/tomgiam/projects/andiamo_places/builds/38a24436-6692-488c-b1fb-cc874c37b68c

### Summary

My project builds fine with "expo build:android -t app-bundle", but fails with "eas build". The error is "Gradle build failed with unknown error. Please see logs for the "Run gradlew" phase." My eas.json is:
{
    "build": {
      "production": {
        "env": {
          "EAS_NO_VCS": "1"
        }
      }
    }
  }

The main reason that I'm trying to use the eas build, is because android no longer likes the result from the expo build. It gives me the error message: "Your app com.andiamoplaces.andiamotravelapp version code 9 includes SDK com.segment.analytics.android:analytics, 4.8.2 or an SDK that one of your libraries depends on", android also complains about SDK com.segment.analytics.android:analytics, 4.9.4. I upgraded from SDK 44 to SDK 45 but the problem did not go away, so I tried the eas build and it doesn't work, so now I'm stuck. I could use the expo build until August 9, 2022, but after that, google may not accept my app or even shut it down.

### Managed or bare?

Managed

### Environment

  expo-env-info 1.0.4 environment info:
    System:
      OS: Windows 10 10.0.19044
    Binaries:
      Node: 16.16.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 7.14.0 - C:\Program Files\nodejs\npm.CMD
    SDKs:
      Android SDK:
        API Levels: 28
        Build Tools: 28.0.3
        System Images: android-28 | Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom  
    IDEs:
      Android Studio: Version  3.6.0.0 AI-192.7142.36.36.6200805
    npmPackages:
      expo: ~45.0.0 => 45.0.6
      react: 17.0.2 => 17.0.2
      react-dom: 17.0.2 => 17.0.2
      react-native: 0.68.2 => 0.68.2
      react-native-web: 0.17.7 => 0.17.7
      react-navigation: ^4.4.4 => 4.4.4
    Expo Workflow: managed

√ Found all copies of expo-modules-autolinking
  All copies of expo-modules-autolinking satisfy ~0.8.1 || ~0.9.0
√ Found all copies of @expo/config-plugins
Expected package @expo/config-plugins@^4.1.0
Found invalid:
  @expo/config-plugins@4.0.7
  @expo/config-plugins@4.0.7
  @expo/config-plugins@4.0.7
  @expo/config-plugins@4.0.7
  (for more info, run: npm why @expo/config-plugins)
√ Found all copies of @expo/prebuild-config
Expected package @expo/prebuild-config@^4.0.0
Found invalid:
  @expo/prebuild-config@3.0.7
  (for more info, run: npm why @expo/prebuild-config)

### Error output

EXPO_DEBUG=true eas build
EXPO_DEBUG=true : The term 'EXPO_DEBUG=true' is not recognized as the name of a cmdlet, function, script file, or operable program. 
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ EXPO_DEBUG=true eas build
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (EXPO_DEBUG=true:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

### Reproducible demo or steps to reproduce from a blank project

eas build
[choose Android]

**Comment by wkozyra95** [comment]
You have incompatible  packages listed in expo doctor

> EXPO_DEBUG=true : The term 'EXPO_DEBUG=true' is not recognized as the name of a cmdlet, function, script file, or operable

This is not how you set env variables  on windows

Specifying `"EAS_NO_VCS": "1"` in eas.json does not do anything, also I don't recommend using that env unless you know what you are doing.

The linked build is failing because some image that is supposed to be a png file is either malformed or different format(e.g. jpg with a png in name)

Closing the issue as the original problem is not eas related and the build errors seem to be a misconfiguration.

**Comment by tomgiam** [comment]
@wkozyra95  Thanks, I was able to get the eas build to work by changing one image to a png format. However, I'm still getting the error message from google "Your app com.andiamoplaces.andiamotravelapp version code 15 includes SDK com.segment.analytics.android:analytics or an SDK that one of your libraries depends on". So, I opened up a new issue for that one. The strange part is that my app is version 20 (1.3.0) and not version 15, so I don't know what the version 15 is referring to.
